zerver/webhooks/gocd/tests.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import urllib
+from typing import Text
+
+from zerver.lib.test_classes import WebhookTestCase
+from zerver.models import get_realm, get_user
+
+class GocdHookTests(WebhookTestCase):
+    STREAM_NAME = 'gocd'
+    URL_TEMPLATE = "/api/v1/external/gocd?stream={stream}&api_key={api_key}&topic=builds"
+    FIXTURE_DIR_NAME = 'gocd'
+    TOPIC = 'builds'
```

Comment:
@Balaji2198: I think the topic should be something else here. Maybe something specific to a build or the repo/branch being tested?